worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -341,16 +343,19 @@ class FetchHandler(
   def handleEndStreamFromClient(streamId: Long, streamType: StreamType): Unit = {
     streamType match {
       case StreamType.ChunkStream =>
+        incChunkAppConnectionCounter(streamId)
```

Comment:
This means that client has closed this logic connection with the server. Shouldn't increase this counter.